src/Command/FetchAssetsCommand.php

```diff
@@ -156,11 +157,19 @@ protected function execute(Console\Input\InputInterface $input, Console\Output\O
         }
 
         if (!$successful && $assetCount > 1) {
-            $this->io->warning('Command finished with errors.');
+            if ($this->hasWarnings) {
+                $this->io->warning('Command finished with errors and warnings.');
+            } else {
+                $this->io->warning('Command finished with errors.');
+            }
```

Comment:
We should avoid nested `if` segments. In this case, we can simplify the whole thing:

```suggestion
            $this->io->warning(sprintf('Command finished with errors%s.', $this->hasWarnings ? ' and warnings' : ''));
```

Reply:
Ugh. Of course.